core/unit_test/CMakeLists.txt

```diff
@@ -835,6 +921,19 @@ if ((KOKKOS_ENABLE_OPENMPTARGET OR KOKKOS_ENABLE_OPENACC) AND KOKKOS_CXX_COMPILE
   )
 endif()
 
+# FIXME_OPENACC - Comment non-passing tests with the Clang compiler
+if (KOKKOS_ENABLE_OPENACC AND KOKKOS_CXX_COMPILER_ID STREQUAL Clang)
+  SET(DEFAULT_DEVICE_SOURCES
+    UnitTestMainInit.cpp
+    TestInitializationSettings.cpp
+    TestParseCmdLineArgsAndEnvVars.cpp
+    #default/TestDefaultDeviceType.cpp
+    default/TestDefaultDeviceType_d.cpp
+    default/TestDefaultDeviceTypeResize.cpp
+    default/TestDefaultDeviceTypeViewAPI.cpp
+  )
```

Comment:
Does `TestCompilerMacros.cpp` really not pass?

I see now that this is how we currently do it but I would really prefer if we expressed it in terms of what tests are not supported (remove the ones that are not currently supported)

Reply:
@rgayatri23 

Please fix https://github.com/kokkos/kokkos/blob/35c1e3c37402ff8505410456421145e0cd50812c/core/unit_test/CMakeLists.txt#L830-L835
and use
```cmake
list(REMOVE_ITEM DEFAULT_DEVICE_SOURCES
  <remove-this>
  <remove-that>
)

Reply:
> Does `TestCompilerMacros.cpp` really not pass?
> 
> I see now that this is how we currently do it but I would really prefer if we expressed it in terms of what tests are not supported (remove the ones that are not currently supported)

`TestCompilerMacros.cpp` passes when compiled by NVHPC V22.11 and Clacc (built on Clang V17.0.0) and thus is re-enabled.